### PR TITLE
SM-7993 Added option to disable oauth proxy for manual auth token handling

### PR DESCRIPTION
**Pull request by simensma**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "url": "git://github.com:MrSwitch/hello.js.git"
   },
   "main": [
-    "dist/hello.all.js"
+    "dist/hello.all.min.js"
```
Comment (blx) [inline bower.json]: Why is this? On our end, grunt should concat and minify all the third party deps at build time already. On hello.js's end, the package file should export the "civilized" unminified version, usually.

File: dist/hello.all.js [hunk 2]
```diff
@@ -5154,6 +5165,37 @@ if (typeof chrome === 'object' && typeof chrome.identity === 'object' && chrome.
 })(hello);
 
 (function(hello) {
+	hello.init({
```
Comment (svaldivia) [inline dist/hello.all.js]: Is this a 2 space tab?

File: src/modules/spotify.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+(function(hello) {
+	hello.init({
+		spotify: {
```
Comment (svaldivia) [inline src/modules/spotify.js]: There is some serious indentation going on here...

**Comment by svaldivia** [comment]
I think my indent comment is just github being weird....

File: src/modules/spotify.js (hunk 3 above)
Comment (simensma) [inline src/modules/spotify.js]: Same as above :)

**Pull request merged**